tree-sitter/src/TreeSitter/Span.hs

```diff
@@ -0,0 +1,18 @@
+module TreeSitter.Span
+( Span(..)
+, Pos(..)
+) where
+
+import Prelude hiding (span)
```

Comment:
I think this import is likely redundant?

Reply:
Word.